lib/sync_dao.es6.js

```diff
@@ -97,15 +101,24 @@ foam.CLASS({
 
         // TODO(markdittmer): Clean this up. Need a way to override lookup and
         // requiresCreate.
-        var lookup = this.lookup.bind(this);
+        const lookup = this.lookup.bind(this);
+
+        const indexedMDAO = lookup('foam.dao.MDAO').create({
+          of: this.of,
+        }, this.container.ctx);
+        // Index releaes ID on release/interface junction DAO.
+        if (this.of === pkg.VersionedReleaseWebInterfaceJunction) {
```

Comment:
Does the MDAO have any benefits (or drawbacks) if you don't hit this codepath? This feels like an OO antipattern, and the Right Way would usually be to subclass for this case.

Reply:
Do you mean implement `VersionedReleaseWebInterfaceJunctionMDAO` with default `of` class and default property index setup?

Reply:
Something like that